crates/typst-library/src/info.rs

```diff
@@ -0,0 +1,19 @@
+//! Information about typst itself
+
+use typst::eval::{Module, Scope, Value, Version, VersionComponent};
+
+/// Construct the module with information about typst itself
+pub fn module() -> Module {
+    let mut scope = Scope::deduplicating();
+
+    let version = Version::from_iter([
+        env!("CARGO_PKG_VERSION_MAJOR").parse::<VersionComponent>().unwrap(),
+        env!("CARGO_PKG_VERSION_MINOR").parse::<VersionComponent>().unwrap(),
+        env!("CARGO_PKG_VERSION_PATCH").parse::<VersionComponent>().unwrap(),
+    ]);
+
+    scope.define("version", Value::Version(version));
+    scope.define("commit", option_env!("TYPST_COMMIT"));
+
+    Module::new("info").with_scope(scope)
```

Comment:
I would prefer `sys`.

Reply:
I'm holding off the bikeshedding for now. I think we should worry about the name as the last thing after everything else has been taken care of.